compress/jbig2_decompress_with_globals.go

```diff
@@ -0,0 +1,85 @@
+package main
```

Comment:
Add comment on top, consistent with other examples

Reply:
Already done for all files.